Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,148 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
+ // Declaring the Variables
 
-        public static int FindDigit(string equation)
-        {
-            // Add your code here.
-            throw new NotImplementedException();
-        }
+        int missingDigit=0, pos, FirstNumber, SecondNumber, ResultNumber, FirstNewNumber, SecondNewNumber, NewResult;
+        string[] stringArrayFirst, stringArraySecond, digitsArray;
+
+        // method 
+
+        public int FindDigit(String equation)

```

Comment:
Why the same code written in two files. FindDigit method should be in class FixMultiplication and should be called Test method by creating an instance of the class.